src/client/Microsoft.Identity.Client/Platforms/iOS/iOSCryptographyManager.cs

```diff
@@ -1,52 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
```

Comment:
good cleanup!

Reply:
That was done by you. https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/commit/d0c782e85b104d7a158150ea5036f72eb5d24f8f